### PR TITLE
Currency: Added build-script and build_info.json for erlang

### PR DESCRIPTION
**Pull request by pooja0805**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

**Comment by pooja0805** [comment]
Verification Log - [ErlangScriptLog.txt](https://github.com/ppc64le/build-scripts/files/11146401/ErlangScriptLog.txt)

File: e/erlang/build_info.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+    "package_name": "erlang",
```
Comment (nikhil-kalbande) [inline e/erlang/build_info.json:2]: Lets add package name as `erlang-otp` in dashboard. I see erlang is Org name

File: e/erlang/build_info.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    "github_url": "https://github.com/erlang/otp.git",
+    "version": "OTP-25.3",
+    "package_dir": "e/erlang/",
+    "build_script": "erlang_ubi_8.5.sh",
```
Comment (nikhil-kalbande) [inline e/erlang/build_info.json:6]: keep erlang-otp in script name also

File: e/erlang/erlang_ubi_8.5.sh [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+fi
+
+#Testing Erlang
+#Build and release the test suites
+make release_tests
+cd release/tests/test_server
+
+if ! $ERL_TOP/bin/erl -s ts install -s ts smoke_test batch -s init stop; then
+        echo "Test Fails"
+	exit 1
```
Comment (nikhil-kalbande) [inline e/erlang/erlang_ubi_8.5.sh:70]: for test make it exit 2

**Review by nikhil-kalbande — CHANGES_REQUESTED** [review]
Also please check PR from Amit if we need to incorporate anything from that. - https://github.com/ppc64le/build-scripts/pull/2988/files

File: e/erlang/build_info.json (hunk 1 above)
Comment (pooja0805) [reply e/erlang/build_info.json:2]: Okay. Will update it.

File: e/erlang/build_info.json (hunk 2 above)
Comment (pooja0805) [reply e/erlang/build_info.json:6]: Okay. Will update it.

File: e/erlang/erlang_ubi_8.5.sh (hunk 3 above)
Comment (pooja0805) [reply e/erlang/erlang_ubi_8.5.sh]: Updated

**Pull request merged**